engine.py

```diff
@@ -1027,7 +1032,6 @@ def __send_state(self):
         }
 
         # send the commands back to adobe
-        self.logger.debug("Sending commands: %s" % str(all_commands))
```

Comment:
Removed these debug calls since the corresponding `self.adobe.send_*` command already logs these. Reduces some of the duplication in the logs.